data/org.freedesktop.impl.portal.InputCapture.xml

```diff
@@ -31,6 +31,119 @@
       This documentation describes version 1 of this interface.
   -->
   <interface name="org.freedesktop.impl.portal.InputCapture">
+    <!--
+        CreateSession:
+        @handle: Object path for the :ref:`org.freedesktop.impl.portal.Request` object representing this call
+        @session_handle: Object path for the :ref:`org.freedesktop.impl.portal.Session` object representing the session being created
+        @app_id: App id of the application
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+        @response: Numeric response
+        @results: Vardict with the results of the call
+
+        Create a capture input session.
+
+        Supported keys in the @options vardict include:
+
+        * ``capabilities`` (``u``)
+
+          Bitmask of requested capabilities, see the
+          SupportedCapabilities property. This value is required.
+
+        The following results get returned in the @results vardict:
+
+        * ``session_id`` (``s``)
+
+          The session id. A string representing the created capture input session.
+
+        * ``capabilities`` (``u``)
+
+          The capabilities available to this session. This is always a
+          subset of the requested capabilities.
+    -->
+    <method name="CreateSession">
```

Comment:
There is already `CreateSession` in this interface, so no need to add this.

Reply:
oh i see thanks updating soon.